### ISSUE TITLE
Missing critical information about YouTubeDebugger

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/react-updating-state-lab#15]
The README doesn't indicate this, but in order for the YouTubeDebugger tests to work, you need to wrap the `<button>`s in a `<div>` or some other HTML element.

Here's what I did initially (without giving away the whole solution):
```
render() {
    return ([
      <button></button>,
      <button></button>
    ])
  }
```
It worked in the Chrome browser, but the test output looked like this:
```
1) <YouTubeDebugger /> should have the correct initial state:
     Error: Method “state” is meant to be run on 1 node. 2 found instead.
      at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1652:13)
      at ShallowWrapper.state (node_modules/enzyme/src/ShallowWrapper.js:1194:25)
      at Context.<anonymous> (test/index-test.js:55:20)
      at processImmediate (internal/timers.js:456:21)

  2) <YouTubeDebugger /> should update the bitrate when the `.bitrate` button is clicked:
     Error: Method “setState” is meant to be run on 1 node. 2 found instead.
      at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1652:13)
      at ShallowWrapper.setState (node_modules/enzyme/src/ShallowWrapper.js:742:10)
      at YouTubeDebugger.ShallowWrapper.instance.setState (node_modules/enzyme/src/ShallowWrapper.js:422:66)
      at YouTubeDebugger._this.changeBitrate (src/components/YouTubeDebugger.js:21:10)
      at node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:716:13
      at withSetStateAllowed (node_modules/enzyme-adapter-utils/src/Utils.js:99:18)
      at Object.simulateEvent (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:712:11)
      at ShallowWrapper.<anonymous> (node_modules/enzyme/src/ShallowWrapper.js:1134:7)
      at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1654:21)
      at ShallowWrapper.simulate (node_modules/enzyme/src/ShallowWrapper.js:1133:17)
      at Context.<anonymous> (test/index-test.js:63:30)
      at processImmediate (internal/timers.js:456:21)

  3) <YouTubeDebugger /> should update the video resolution when the `.resolution` button is clicked:
     Error: Method “setState” is meant to be run on 1 node. 2 found instead.
      at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1652:13)
      at ShallowWrapper.setState (node_modules/enzyme/src/ShallowWrapper.js:742:10)
      at YouTubeDebugger.ShallowWrapper.instance.setState (node_modules/enzyme/src/ShallowWrapper.js:422:66)
      at YouTubeDebugger._this.changeVideoResolution (src/components/YouTubeDebugger.js:32:10)
      at node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:716:13
      at withSetStateAllowed (node_modules/enzyme-adapter-utils/src/Utils.js:99:18)
      at Object.simulateEvent (node_modules/enzyme-adapter-react-16/src/ReactSixteenAdapter.js:712:11)
      at ShallowWrapper.<anonymous> (node_modules/enzyme/src/ShallowWrapper.js:1134:7)
      at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1654:21)
      at ShallowWrapper.simulate (node_modules/enzyme/src/ShallowWrapper.js:1133:17)
      at Context.<anonymous> (test/index-test.js:73:33)
      at processImmediate (internal/timers.js:456:21)
```
In order to get the tests passing, I had to wrap the `<button>` elements in a `<div>`:
```
render() {
    return (
      <div>
        <button></button>
        <button></button>
      </div>
    )
  }
```
As I said above, though, the README doesn't tell us to do this at all.

Thanks, as always, for looking into this!

Sdcrouse

**Comment by graciemcguire** [comment]
Thanks for your PR and for raising this issue, @Sdcrouse !

We merged in your first typo fix, and added some changes to the your note related to this issue. Remember,  all React components must act like pure functions with respect to their props, and thus can only return one element. Returning multiple elements can be accomplished by wrapping them in a wrapper element. We made some changes to your text, as it was specific to this lab and not all components in general. 

Thank you for raising this issue, and happy coding!
